muninforAndroid/src/main/res/values/strings.xml

```diff
@@ -222,30 +224,30 @@
 	<string name="move_down">Move down</string>
 	<string name="move_left">Move left</string>
 	<string name="rate">Rate the app</string>
-	<string name="rate_long">If you like the app, would you mind rating it on Google Play? Thank you!</string>
+	<string name="rate_long">If you like the app, rate it on Google Play. Thank you.</string>
```

Comment:
It sounds like begging. From a neutral and considering the psychological aspect:
"Your enjoyment of this app is needed in rating this app, which could help others in forming their decisions." Though it is much longer.

Reply:
I really like this one!
I didn't quite like "If you like the app, rate it on Google Play. Thank you." because it looks like an imperative & too much informal statement (the two dots makes it weird IMHO)